platform/commonUI/edit/bundle.js

```diff
@@ -203,20 +205,30 @@ define([
                     ]
                 },
                 {
+                    "key": "save-and-stop-editing",
+                    "category": "save",
+                    "implementation": SaveAndStopEditingAction,
+                    "name": "Save and Done Editing",
```

Comment:
lol. 

@BogdanAlexandru The consensus is "Save and Finish" 😃 Sorry for being so collectively indecisive.